### PR TITLE
Add Gitter Sidecar to home page

### PR DESCRIPTION
**Pull request by MattMS**
May allow #100 to be closed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -142,5 +142,12 @@ <h2 id="acknowledgements">Acknowledgements</h2>
 <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/">CC BY-NC-SA 3.0</a>.</p>
 </article>
     </main>
+
+    <script>
+        ((window.gitter = {}).chat = {}).options = {
+        room: 'ramda/ramda'
```
Comment (buzzdecafe) [inline index.html]: why not simply `window.gitter = {chat: {options: {room: 'ramda/ramda'}}};`?

**Comment by buzzdecafe** [comment]
i think this is fine for the index.html, but i'd also like people to be able to get directly from the docs pages to the gitter room. i don't think the sidecar tray is a good fit on the docs pages, though. i think a nav link may be a good idea as well

**Comment by MattMS** [comment]
No worries. I wasn't sure if it did fulfil #100 which is why I tried to avoid the magic closing text in the summary.

Happy to make another pull request to add the link to the header nav.

**Comment by buzzdecafe** [comment]
this one is fine, though pending simple fix above

**Comment by buzzdecafe** [comment]
🐄

**Pull request merged**